### DISCUSSION TITLE
How to get the Pair Extraordinaire badge

### DISCUSSION THREAD

**Discussion by Hezethegamer** [drknzz/GitHub-Achievements#259] [Q&A] [answered]
how do i get Pair Extraordinaire badge

**Comment by the1Riddle** [comment, marked as answer]
you need to be co-authored in a project

**Comment by SUDEEP-M-SHETTY** [comment]
Go to this repository
https://github.com/Prakash4844/Get-Pair-Extraordinaire
Follow the steps provided in the repository and you'll have the badge in no time.
This is the easiest way you can obtain it.
